src/esi/Libxml2Parser.cc

```diff
@@ -16,8 +16,30 @@
 
 #if USE_SQUID_ESI && HAVE_LIBXML2
 
+#include "base/RunnersRegistry.h"
 #include "esi/Libxml2Parser.h"
 
+#include <memory>
+
+namespace Esi
+{
+
+class Xml2Rr : public RegisteredRunner
```

Comment:
AFAICT, this class should be called Libxml2Rr for consistency sake.

Reply:
Done in cbc3ab2